load-test/prep-up.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+# exit on error
+set -e
+
+### Azure CLI script to initialise load-test environment with fresh database
+# Create Database with unique name
+# Bind replica (optional)
+# Run Migrations
+# Seed Data
+# Update admin app database name setting
+# Enable / Disable admin app replica read mode
+# Update function consumption app database name setting
+# Update function app service database name setting
+# Run load test (manual operation)
+# Delete Database
+
+### script arguments
+# $1 main resource group
+# $2 functions resource group
+# $3 sql server
+# $4 db scale (S0, S1 etc.)
+# $5 admin web app instance name
+# $6 function consumption instance name
+# $7 function app service instance name
+# $8 sql replica server name
+
+RES_GROUP=$1
+RES_GROUP_FUNCTIONS=$2
+SQL_SERVER=$3
+DB_SCALE=$4
+ADMIN_APP=$5
+FUNC_CONSUMP=$6
+FUNC_APPSVC=$7
+SQL_SERVER_REPLICA=$8
+
+# Create database with unique name
+DB_SUFFIX=$(openssl rand -hex 4)
```

Comment:
Is `openssl` guaranteed to be installed?  What happens if it isn't?

Reply:
that's a very good point....

Reply:
the idea is that this will be run on a testers/developers machine, so i think we can just demand it as a requirement...

Reply:
That's fine then 👍